core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,34 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    # This phase is used to give time to Cromwell to abort all workflows upon shutdown. It's only used if system.abort-jobs-on-terminate = true
+    abort-all-workflows {
+      # /!\ This depends-on field should NOT be modified
+      depends-on = [service-unbind]
```

Comment:
I think you're starting to convince me :) I'm fine with putting things in the conf and I don't like having "config strings" hardcoded anyway. If we go that route I think we could at least ping workbench and gotc to make sure they're aware that the *must* include `application.conf` and *must not* mess around with those shutdown values